.github/workflows/check-contrib.yml

```diff
@@ -0,0 +1,83 @@
+name: check-contrib
+
+on: [push, pull_request]  # would you need to define PRs containing .eb files?
+
+permissions:
+  contents: read  # to fetch code (actions/checkout)
+
+jobs:
+  check-contrib:
+    name: Run EasyBuild check-contrib
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2  # shallow but deep enough to diff last commit; I don't want to have do to fetch-depth: 0 and get the full history
+
+      - name: Set up Python env
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.8'
+
+      - name: Upgrade pip
+        run: python -m pip install --upgrade pip
+
+      - name: Install EasyBuild and required deps
```

Comment:
Add caching for Python dependencies using actions/cache@v3 to speed up subsequent runs.